circle.yml

```diff
@@ -2,10 +2,8 @@ machine:
   services:
     - docker
   environment:
-    GODIST: "go1.8.linux-amd64.tar.gz"
+    GODIST: "go1.9.linux-amd64.tar.gz"
```

Comment:
At some point I think we'd really like to test this against a set of different versions (say, `1.7` to `1.9`) but let's save this for later.